content/zh-cn/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -33,7 +33,7 @@ in this task demonstrate a strategic merge patch and a JSON merge patch.
 Here's the configuration file for a Deployment that has two replicas. Each replica
 is a Pod that has one container:
 -->
-## 使用策略合并 patch 更新 Deployment    {#use-a-strategic-merge-patch-to-update-a-deployment}
+## 使用策略合并 patch 更新 Deployment {#use-a-strategic-merge-patch-to-update-a-deployment}
```

Comment:
this kind of changes are unnecessary.

The extra spaces (4-6) are for readability's sake.

Reply:
Thanks for you time , Ihave remove these changes.